docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -108,6 +108,8 @@ API Changes with Deprecations
   argument for capability during the deprecation period.
 * Parameter `pvsystem.DC_MODEL_PARAMS` is renamed to `pvsystem._DC_MODEL_PARAMS`.
   Users should not rely on this dictionary's existence or structure.
+* The `modelchain.ModelChain.diode_params` attribute is now formatted in
+  a pandas.DataFrame with `DatetimeIndex`, rather than in a tuple.
```

Comment:
This is currently in the `API Changes with Deprecations` section of the whats new file. It should be moved to the `API Breaking Changes` section.